### PR TITLE
Add GitHub workflow for Brim releases

### PR DESCRIPTION
**Pull request by nwt**
When a tag matching `/^v.*brim.*$/` is pushed, the workflow creates a
GitHub release with a macOS artifact called zeek-TAG.darwin-amd64.zip
with these contents:

    zeek/
      bin/zeek           Zeek executable
      share/zeek/base/   Zeek base scripts
      share/zeek/policy/ Zeek policy scripts
      share/zeek/site/   Zeek site scripts
      zeek               executable shell script to set ZEEKPATH and
                         ZEEK_PLUGIN_PATH and then exec bin/zeek

### CODE REVIEW

File: .github/workflows/brim-release.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+jobs:
+  macos:
+    runs-on: macos-10.15
+    steps:
+      - uses: actions/checkout@v2
+      - run: git submodule update --init --recursive
+      - run: brew install bison openssl
+      # Prefer static libraries.  Appending to this file is hacky but effective.
+      - run: echo 'set(CMAKE_FIND_LIBRARY_SUFFIXES .a ${CMAKE_FIND_LIBRARY_SUFFIXES})' >> cmake/CommonCMakeConfig.cmake
```
Comment (nwt) [inline .github/workflows/brim-release.yml]: I did this to keep CMakeList.txt out of this change, but if it strikes anyone as too dicey, we could instead add the `set(...)` to that file.

**Comment by nwt** [comment]
@mikesbrown:I triggered https://github.com/brimsec/zeek/actions/runs/46998542 by pushing tag vBrimReleaseWorkflowTest-brim.

File: .github/workflows/brim-release.yml (hunk 1 above)
Comment (henridf) [reply .github/workflows/brim-release.yml]: We can do this after I move henridf/cmake to brimsec/cmake (and try to contribute it to upstream zeek and cmake, possibly enabled by a configure flag?).

Comment (alfred-landrum) [reply .github/workflows/brim-release.yml]: Would you please add the changes to the CMakeList.txt, along with a comment like: `# Brim: Prefer static ...`.

File: .github/workflows/brim-zeek-wrapper [hunk 2]
```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
```
Comment (alfred-landrum) [inline .github/workflows/brim-zeek-wrapper]: Would you please move this script into the top level `brim` folder I mentioned in my earlier comment?

File: .github/workflows/brim-release.yml (hunk 1 above)
Comment (nwt) [reply .github/workflows/brim-release.yml]: 👍

File: .github/workflows/brim-zeek-wrapper (hunk 2 above)
Comment (nwt) [reply .github/workflows/brim-zeek-wrapper]: 👍

File: .github/workflows/brim-release.yml [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+  push:
+    tags:
+      - v*brim*
+
+jobs:
+  macos:
+    runs-on: macos-10.15
+    steps:
+      - uses: actions/checkout@v2
+      - run: ./brim/release
```
Comment (alfred-landrum) [inline .github/workflows/brim-release.yml]: nit: Would you put the string `macos` somewhere in the release name? It calls out to `brew`, so it's mac specific.

File: brim/zeek [hunk 4]
```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+file=$(realpath "${BASH_SOURCE[0]}")
```
Comment (alfred-landrum) [inline brim/zeek]: I don't have `realpath` in my path on my mac; I think that's from gnu coreutils? I'm used to seeing it on linux.

**Pull request merged**